helm/default-apps-openstack/templates/calico.yaml

```
@@ -0,0 +1,15 @@
+apiVersion: application.giantswarm.io/v1alpha1
+kind: App
+metadata:
+  annotations:
+    chart-operator.giantswarm.io/force-helm-upgrade: "false"
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: calico-{{ .Values.cluster.name }}
```

```suggestion
  name: {{ .Values.cluster.name }}-calico
```

@MarcelMue @tfussell Should the cluster name be the suffix instead of the prefix?

If you do a `kubectl get apps` without any label selectors I think that order is more useful. WDYT?

---

I think making it a suffix makes sense - ordered by cluster is nice.

---

Done!